FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -1,7 +1,108 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MemberInfoCard.module.scss';
+import Card from 'react-bootstrap/Card';
+import { contactIcons } from '@icons';
+import { Link, FilledImage } from '@basics';
+import { Body1, Body2, Caption } from '@components/basics';
+import cn from 'classnames';
 
-const MemberInfoCard: FunctionComponent = () => <Container></Container>;
+export interface MemberInfoCardProps {
+  name: string;
+  imgSrc: string;
+  title: string;
+  quote: string | { number: number; length: string };
+  website?: string;
+  email?: string;
+  linkedIn?: string;
+}
+
+const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
+  name,
+  imgSrc,
+  title,
+  quote,
+  website,
+  email,
+  linkedIn,
+}) => {
+  //establishing a first name for use in text bodeis
+  var nameArray: string[] = name.split(' ');
+  var firstName: string = nameArray[0];
+  //changing the grammar for design team
+  var team: string = title.toLowerCase();
+  team === 'designer' ? (team = 'design') : null;
+
+  //left side of the card, consists of picture, name, title, and quote
```

Comment:
Comments should explain "why" and not "what" or "how". Code should explain "what" or "how". If the code is unclear without this comment, then the code should be written more clearly. Maybe consider if you can improve the var name or if you can split it up further into smaller components to make it more readable. I'll leave it up to you how to do this, reach out if you need help